client/src/style.scss

```diff
@@ -59,22 +32,22 @@ html {
 }
 
 #g-carousels {
-  font-family: $font-regular;
+  font-family: "Noto Sans";
   font-size: 16px;
   width: 97vw;
   max-width: 1200px;
   min-width: 400px;
   height: auto;
-  margin: 0 auto;
+  margin: 4rem auto;
   color: $font-color;
   line-height: 1.5;
   .g-carousel-title {
     line-height: 1.3334;
     letter-spacing: -0.5px;
     text-align: center;
-    font-family: $font-bold;
+    font-weight: bold;
```

Comment:
I believe on IKEA's site they used 400 for regular and 700 for bold--I'm not sure how different these weights are from the ones CSS assigns (based on our API request, it may be that it's only importing 400 and 700 as options so CSS will just choose the closest--I honestly don't know how exactly it handles that)